api/envoy/extensions/access_loggers/open_telemetry/v3alpha/logs_service.proto

```diff
@@ -0,0 +1,42 @@
+syntax = "proto3";
+
+package envoy.extensions.access_loggers.open_telemetry.v3alpha;
+
+import "envoy/extensions/access_loggers/grpc/v3/als.proto";
+
+import "opentelemetry/proto/common/v1/common.proto";
+
+import "udpa/annotations/status.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.access_loggers.open_telemetry.v3alpha";
+option java_outer_classname = "LogsServiceProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).work_in_progress = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: OpenTelemetry (gRPC) Access Log]
+
+// Configuration for the built-in *envoy.access_loggers.open_telemetry*
+// :ref:`AccessLog <envoy_api_msg_config.accesslog.v3alpha.AccessLog>`. This configuration will
```

Comment:
this link *needs* to be `envoy_v3_api_msg_config.accesslog.v3.AccessLog`

Reply:
it seems like there is some mangling that happens - checking the file proto the line is:

```
  21 │ // Custom configuration for an :ref:`AccessLog <envoy_api_msg_config.accesslog.v3.AccessLog>`

```

so it seems like `envoy_api_msg_config` is translated to `envoy_v3_api_msg_config` which is the actual link prefix

either way the ending needs to be `v3.AccessLog` in all cases for the v3 protos

Reply:
Changed to v3.
FYI for v4alpha it's set to v4alpha.AccessLog in other access loggers as well.

Reply:
yep - cool - i think it works ok because they arent yet built - and should magically work when they are